projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/impala_connection.py

```diff
@@ -52,7 +52,7 @@ def _connect(self):
             host=self._host,
             port=self._port,
             database=self._database,
-            timeout=self._timeout,
+            timeout=int(self._timeout) if self._timeout else None,
```

Comment:
Out of curiosity, the timeout is set through an env var, but what would happen if a user mistypes the value and adds a symbol like a `,` by mistake? The interpreter would generally throw a ValueError, but would we have a clear way of showing that to users?

Reply:
Doesn't seem like it, right now we get:
```
Retrying vdk.internal.builtin_plugins.connection.managed_connection_base.ManagedConnectionBase.connect in 60.0 seconds as it raised ValueError: invalid literal for int() with base 10: '4,'.
```
and this is only in the debug logs, as far as a normal user is concerned VDK is frozen.

Reply:
Eventually it times out with `ValueError: invalid literal for int() with base 10: '4,`.

Reply:
I see. Well, the fact that these are visible only in debug logs is probably not the best UX, but I guess this could be a topic for another discussion.

Reply:
The type of the variable is determined by the default value - https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/impala_plugin.py#L54 

Maybe we can set default value '-1' or '0' ?